archr/targets/docker_target.py

```diff
@@ -315,7 +314,6 @@ def check_in_docker() -> bool:
     with open("/proc/1/cgroup", "r") as f:
         return "docker" in f.read()
 
-
```

Comment:
There should be two empty lines here

Reply:
OK

Reply:
wait I don't get it. Shouldn't the double empty lines be above `check_in_docker` not below?

Reply:
There should be two above and two below. There should be two empty lines between each top-level statement.

Reply:
OK. changed